interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,212 @@
+
+- Data preparation including importing, validating and cleaning, munging and transformation, normalization, and staging
+- Training configuration including parameterizing arguments, filepaths, and logging / reporting configurations
+- Training and validating efficiently and repeatedly. Efficiency might come from specifying specific data subsets, different hardware compute resources, distributed processing, and progress monitoring
+- Deployment, including versioning, scaling, provisioning, and access control
+
+An Azure ML pipeline performs a complete logical workflow with an ordered sequence of steps. Each step is a discrete processing action. Pipelines run in the context of an Azure Machine Learning Experiment.
+In the early stages of an ML project, it's fine to have a single Jupyter notebook or Python script that does all the work of Azure workspace and resource configuration, data preparation, run configuration, training, and validation. But just as functions and classes quickly become preferable to a single imperative block of code, ML workflows quickly become preferable to a monolithic notebook or script.
+By modularizing ML tasks, pipelines support the Computer Science imperative that a component should "do (only) one thing well." Modularity is clearly vital to project success when programming in teams, but even when working alone, even a small ML project involves separate tasks, each with a good amount of complexity. Tasks include: workspace configuration and data access, data preparation, model definition and configuration, and deployment. While the outputs of one or more tasks form the inputs to another, the exact implementation details of any one task are, at best, irrelevant distractions in the next. At worst, the computational state of one task can cause a bug in another.
+
+There are many ways to leverage AzureML. Currently DeepSeismic has integrated with AzureML to train a pipeline, which will include creating an experiment titled "DEV-train-pipeline" which will contain all training runs, associated logs, and the ability to navigate seemlessly through this information. AzureML will take data from a blob storage account and the associated models will be saved to this account upon completeion of the run.
+
+Please refer to microsoft docs for additional information on AzureML pipelines and related capabilities https://docs.microsoft.com/en-us/azure/machine-learning/concept-ml-pipelines
+
+## Files needed for this AzureML run
+
+You will need the following files to complete an run in AzureML
+
+- [.azureml/config.json](../../../.azureml.example/config.json) This is used to import your subscription, resource group, and AzureML workspace
```

Comment:
Could you please include info on how to get one? It's being downloaded from the Portal. Or link documentation. I would just say "For detailed information about using Azure ML Workspace, please see [](); if you're already familiar with Azure ML Workspace, you can download config.json from Azure ML Workspace in the [Portal](portal.azure.com)"

Reply:
Also, according to the code later, I think config.json is to be placed in the repo root directory, but I don't see any instructions about that here

Reply:
So this here is just a list of what is needed for the run as a highlight, I can add additional detail down below where it is mentioned. How does that sound? @maxkazmsft

Reply:
Added information on how to download config.json from the portal in this commit: https://github.com/microsoft/seismic-deeplearning/pull/195/commits/04343e6af970a0d1a933103526b2e104cec5e56b